src/ReactiveUI/ObservableAsPropertyHelper.cs

```diff
@@ -229,11 +229,11 @@ public static class OAPHCreationHelperMixin
         /// An initialized ObservableAsPropertyHelper; use this as the backing field 
         /// for your property.
         /// </returns>
-        public static ObservableAsPropertyHelper<TRet> ToProperty<TObj, TRet>(
-            this IObservable<TRet> This,
+        public static ObservableAsPropertyHelper<TObs> ToProperty<TObj, TProp, TObs>(
```

Comment:
I think it would be worth adding XMLDOC comments for the type params now they do more.